api/types/swarm/swarm.go

```diff
@@ -153,6 +158,8 @@ type InitRequest struct {
 	Spec             Spec
 	AutoLockManagers bool
 	Availability     NodeAvailability
+	DefaultAddrPool  []*net.IPNet
+	SubnetSize       int
```

Comment:
Same here; why `int` instead of `uint32` (if that's what we want?)

Reply:
modified it